src/main/java/org/folio/settings/server/storage/SettingsStorage.java

```diff
@@ -85,9 +85,9 @@ public Future<Void> init() {
             + ")",
         // need two as userId = NULL would be considered unique
         "CREATE UNIQUE INDEX IF NOT EXISTS settings_scope_key_users ON "
-            + settingsTable + "(scope, key, userId) WHERE userId is NOT NULL",
+            + settingsTable + "(scope, key text_pattern_ops, userId) WHERE userId is NOT NULL",
```

Comment:
No comma needed between `key` and `text_pattern_ops`? (I don't know the relevant Postgres syntax, so probably this is fine.)

Reply:
https://www.postgresql.org/docs/current/indexes-opclass.html

Reply:
Thanks!